src/Pororoca.Desktop/PororocaThemeManager.cs

```diff
@@ -72,6 +76,7 @@ public static ISolidColorBrush MapPororocaVariableHighlightBrush(PororocaTheme t
         theme switch
         {
             PororocaTheme.Light => Brushes.ForestGreen,
+            PororocaTheme.Light2 => Brushes.ForestGreen,
```

Comment:
Feel free to change the variable highlight color if you want!

Reply:
After trying around a few colours I think Goldenrod feels pretty right :) 

![image](https://github.com/user-attachments/assets/f6621ea8-0510-4de0-8fdb-33ea11260e87)

Reply:
The contrast checker tool says Goldenrod is not clear enough :-(

Reply:
@lilacse

Reply:
Oops, didn't see this comment... I'll find another colour after I get back from work today!

Reply:
MediumVioletRed fits quite well into the UI too imo, and the contrast looks healthy as well. 

![image](https://github.com/user-attachments/assets/b6f1c68d-bbbc-4b51-b8c2-e4e5fc140389)

When you mentioned about contrast, I recalled that many light themes across popular IDEs would fail the contrast test in many ways... although some of them are forgivable, depending on what are the elements that aren't clear enough. But since we are highlighting user variables, I think it's indeed a good idea to choose a colour that looks clearer.